.github/workflows/docs-quality-checker.yml

```diff
@@ -0,0 +1,37 @@
+name: Check Markdown files quality
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+jobs:
+  check-all-files:
+    if: github.event_name == 'push'
```

Comment:
follow-up from the comment above - so we want the documentation quality check to always be ran on all files (not only on changed files), on pull requests.

but from my previous comment, you can add paths on when the action should get triggered. See docs here https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#example-including-paths

So I think in this case that will be something like:

```
on:
  pull_request:
    branches: [master]
    paths:
    - '**.md'
```

Reply:
Thanks!